.travis.yml

```diff
@@ -127,4 +127,4 @@ script:
           --volume=`pwd`:/go/src/go.etcd.io/etcd gcr.io/etcd-development/etcd-test:go${TRAVIS_GO_VERSION} \
           /bin/bash -c "GOARCH=386 PASSES='unit' ./test"
         ;;
-    esac
+    esac
```

Comment:
you need to include the s390x cross platform build in this file to enable the test in ci.